yang/IETF-04/ietf-vpn-common.yang

```diff
@@ -575,24 +575,28 @@ module ietf-vpn-common {
     base protocol-type;
     description
       "Transport based on SR.";
-    reference
-      "";
+      reference 
```

Comment:
We will need to add all these references to the Informative References of the draft and call them in the description text.

Reply:
Updated